.github/workflows/CI.yml

```
@@ -39,4 +39,5 @@ jobs:
       if: matrix.os != 'windows-latest'
       run: |
           # Task to auto update version to the next development iteration
-          ./gradlew updateVersion -DnewVersion=2.15.0-SNAPSHOT
+          ./gradlew updateVersion -DnewVersion=2.15.0-SNAPSHOT && ./gradlew build
```

you could run this in one go (you're anyway not doing it in two GHA steps, so you can as well combine the two gradle steps. WDYT?
```suggestion
          ./gradlew updateVersion build -DnewVersion=2.15.0-SNAPSHOT
```

---

No, not really: `./gradlew updateVersion -DnewVersion=2.15.0-SNAPSHOT` modifies build file and if add `build`, the non-modified version will be used, not what we need

---

ah, damn - good catch!